README.md

```diff
@@ -7,7 +7,34 @@ Map engine to serve all the people ;)
 
 ## Setup
 
-Use python 3.10, install poetry: `pip install poetry` and then install dependencies: `poetry install`
+#### 1. Use python 3.10
+If you have a different version of Python on your system, install python 3.10 alongside. For that, you can use [`pyenv`](https://github.com/pyenv/pyenv). Follow the [documentation](https://github.com/pyenv/pyenv?tab=readme-ov-file#installation). Useful commands: `pyenv install`, `pyenv shell`, `pyenv versions`.
+
+#### 2. Set up `poetry`
+`poetry` can create virtual environments associated with a project
+- make sure you are in the Python 3.10 environment.
+- Install `poetry`
+
+```
+pip install poetry
+```
+- Create `poetry` virtual env
+```
+poetry env use
+```
```

Comment:
You are right, this command is not necessary, because `poetry install` creates a virtualenv when it is not present already.
For clarity, `poetry -h env use` says:
```
Description:
   Activates or creates a new virtualenv for the current project.
```